helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/util/TestContext.java

```diff
@@ -1,4 +1,4 @@
-package org.symphonyoss.symphony.bots.helpdesk.bot.it;
+package org.symphonyoss.symphony.bots.helpdesk.bot.util;
```

Comment:
It should be part of test source code or renamed.

Reply:
Renamed!